sdks/python/apache_beam/testing/load_tests/pardo_test.py

```diff
@@ -147,10 +151,18 @@ def process(self, element, state=state_param):
               CounterOperation(
                   self.number_of_counters, self.number_of_operations)))
 
+    if self.window:
+      pc = (
+          pc
+          | beam.WindowInto(beam.window.FixedWindows(1))
+          | beam.GroupByKey())
```

Comment:
Just out of curiosity, could you explain, why do we need GroupByKey in this place?

Reply:
The goal here is to test GroupByKey which holds back windowed values based on the watermark. This creates additional latency. If we do not use GroupByKey, then there is a simple pass-through of all elements without any buffer.

Reply:
I see. In this case, maybe it'd would worth to measure latency in GroupByKey Load Test: https://github.com/apache/beam/blob/master/sdks/python/apache_beam/testing/load_tests/group_by_key_test.py? GroupByKey step is already there.

Reply:
Good point, I think we could move it there.

Reply:
I've removed the GroupByKey here and added a latency measurement to the existing GroupByKey test.